python/lsst/afw/image/image/multiband.py

```diff
@@ -484,10 +484,31 @@ def removeMaskPlane(name):
 
         Parameters
         ----------
-        name: `str`
-            Name of the mask plane to remove
+        name : `str`
+           Name of the mask plane to remove
         """
-        Mask[MaskPixel].removeMaskPlaneDict()
+        Mask[MaskPixel].removeMaskPlane(name)
+
+    def removeAndClearMaskPlane(self, name, removeFromDefault=False):
+        """Remove and clear a mask from the mask plane
+
+        Clear all pixels of the specified mask and remove the plane from the
+        mask plane dictionary.  Also optionally remove the plane from the
+        default dictionary.
+
+        Parameters
+        ----------
+        name : `str`
+           Name of the mask plane to remove
+        removeFromDefault : `bool`, optional
+           Whether to remove the mask plane from the default dictionary.
+           Default is `False`.
+        """
+        # Clear all masks in MultibandMask but leave in default dict for now
+        for single in self.singles:
+            single.removeAndClearMaskPlane(name, False)
```

Comment:
I think `single.removeAndClearMaskPlane(name, removeFromDefault=False)` would be more clear.